client/context/context.go

```diff
@@ -57,9 +62,40 @@ func NewCLIContext() CLIContext {
 		Async:           viper.GetBool(client.FlagAsync),
 		JSON:            viper.GetBool(client.FlagJson),
 		PrintResponse:   viper.GetBool(client.FlagPrintResponse),
+		Certifier:       createCertifier(),
 	}
 }
 
+func createCertifier() tmlite.Certifier {
+	trustNode := viper.GetBool(client.FlagTrustNode)
+	if !trustNode {
```

Comment:
Slightly easier to read: `if trustNode { return nil } else { ... create certifier }`